automation/scripts/local.py

```diff
@@ -15,7 +14,7 @@ def main():
     parser.add_argument("action", choices=["poll"])
     parser.add_argument(
         "--table",
-        type=lambda val: tables.Table[val.upper()],
+        type=lambda val: getattr(tables, val.upper()),
```

Comment:
This also looks sort of odd at the moment. I think we can fix up how tables and their names are registered in the tables module soon (or I can put back the enum).